### PR TITLE
Lines of code

### PR DESCRIPTION
**Pull request by dkrutskikh**
### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix
[ ] New rule
[ ] Changes an existing rule
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain: fix issue #99

### CODE REVIEW

File: doc/metrics/lines-of-executable-code.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Lines of Executable Code
+
+**Lines of Executable Code** is the approximate number of executable code lines or operations in a function of a method. Blank or comment lines are not counted. Function with high values of this metric are often complex and tough to maintain.
```
Comment (Anrock) [inline doc/metrics/lines-of-executable-code.md:3]: > in a function of a method

in a function **or** a method, probably?

File: bin/metrics.dart [hunk 2]
```diff
@@ -28,7 +28,7 @@ Future<void> main(List<String> args) async {
         arguments.rest,
         arguments[ignoredFilesName] as String,
         int.tryParse(arguments[cyclomaticComplexityThreshold] as String ?? ''),
-        int.tryParse(arguments[linesOfCodeThreshold] as String ?? ''),
+        int.tryParse(arguments[linesOfExecutableCodeThreshold] as String ?? ''),
```
Comment (Anrock) [inline bin/metrics.dart:31]: That's a breaking change. Parse old key too to avoid.

File: lib/src/cli/arguments_parser.dart [hunk 3]
```diff
@@ -32,10 +32,10 @@ ArgParser argumentsParser() => ArgParser()
       _printInvalidArgumentValue(cyclomaticComplexityThreshold, i);
     }
   })
-  ..addOption(linesOfCodeThreshold, help: 'Lines of code threshold', valueHelp: '$linesOfCodeDefaultWarningLevel',
```
Comment (Anrock) [inline lib/src/cli/arguments_parser.dart:35]: Removing this is a breaking change.

File: doc/metrics/lines-of-executable-code.md (hunk 1 above)
Comment (dkrutskikh) [reply doc/metrics/lines-of-executable-code.md]: @Anrock fixed

**Review by incendial — APPROVED** [review]

**Pull request merged**